db/dbtypes/types.go

```diff
@@ -88,6 +88,14 @@ const (
 	PoolStatusMissed
 )
 
+type VoteChoice uint8
+
+const (
+	Yes VoteChoice = iota + 1
```

Comment:
why "+ 1"?

Reply:
Wanted to use 0 as the default value that is returned when matching fails

Reply:
I see.  See my comment about `VoteChoiceUnknown`

Reply:
already added `VoteChoiceUnknown` instead of zero